zerver/openapi/zulip.yaml

```diff
@@ -282,6 +282,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonSuccess'
+  /attachments:
+    get:
+      description: Get all the attachments of the requesting user.
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/JsonSuccess'
+                - properties:
+                    attachments:
+                      type: array
+                      description: A list of `attachment` objects, each containing
+                        details about an attachment belonging to the user.
+                    upload_space_used:
+                      type: integer
+                      description: Used upload space by the user in bytes.
+                - example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "attachments": [
+                          {
+                            "id": 1,
+                            "name": "test.txt",
+                            "path_id": "foo/bar/test.txt",
```

Comment:
Are these realistic values?

Reply:
@timabbott These are the values used in the tests for the endpoint in test_attachments.py.

Reply:
Yeah, that's not the right process.

We should be using the actual API in a development environment (upload a file, post a message linking it, and then use the endpoint) to get realistic values.  Otherwise, we risk confusing users with what is potentially hacky test data.

Reply:
@timabbott Thanks for the feedback. I'll keep that in mind.